PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -378,6 +378,18 @@ void AliEmcalJetTask::ExecOnce()
   }
 
   fJetsName = AliJetContainer::GenerateJetName(fJetType, fJetAlgo, fRecombScheme, fRadius, GetParticleContainer(0), GetClusterContainer(0), fJetsTag);
+  std::cout << GetName() << ": Name of the jet container: " << fJetsName << std::endl;
```

Comment:
Would this be better as AliInfoStream()? (Same for below) Or is this because you want it to show up on the train?

Reply:
Indeed AliInfoStream would be the appropriate solution. As the lego train system will everything with a level lower than kError (hard coded in AliAnalysisManager!) this message will not appear. However I want it to appear, as it makes debugging easier in the lego train test and comes with little cost (only displayed once in ExecOnce) the only way to have this would be the usage of std stream (which violates the approach of having a dedicated logging handler for a project) or AliErrorStream (which is a completely inappropriate log level and therefor unlogical) - so I think using std stream is the less inappropriate of the two.

Reply:
Yeah, sounds good - I figured that was the case, but wanted to confirm